src/main/java/ReviewResponse.java

```diff
@@ -13,6 +16,8 @@ public class ReviewResponse {
     @Getter
     String okbb;
 
+    @Setter
+    String blabla;
```

Comment:
Using Lombok annotations like @Setter and @Getter can reduce code readability and make it harder to understand the class's behavior. (Severity: low)